Marlin/src/inc/Conditionals_adv.h

```diff
@@ -1148,7 +1148,7 @@
   #elif HAS_DRIVER(A4988)
     #define MINIMUM_STEPPER_POST_DIR_DELAY 200
   #elif HAS_TRINAMIC_CONFIG || HAS_TRINAMIC_STANDALONE
-    #define MINIMUM_STEPPER_POST_DIR_DELAY 70
+    #define MINIMUM_STEPPER_POST_DIR_DELAY 100
```

Comment:
Is this change on purpose or is it just a leftover? Same for the change in `stepper.cpp` line 3563